saleor/graphql/schema.graphql

```diff
@@ -2158,7 +2158,7 @@ type GiftCard implements Node & ObjectWithMetadata {
   app: App
   expiryPeriod: TimePeriod
   product: Product
-  events: [GiftCardEvent]
+  events: [GiftCardEvent!]
```

Comment:
I think the list should be required too: `[GiftCardEvent!]!` - I guess we never return null for empty lists?